plugins/in_systemd/systemd.c

```diff
@@ -277,14 +283,27 @@ static int in_systemd_collect(struct flb_input_instance *i_ins,
         ctx->pending_records = FLB_FALSE;
         return FLB_SYSTEMD_OK;
     }
-
+    else if (ret_j > 0) {
     /*
-     * ret_j == -1, the loop was broken due to some special condition like
+     * ret_j == 1, but the loop was broken due to some special condition like
      * buffer size limit or it reach the max number of rows that it supposed to
      * process on this call. Assume there are pending records.
      */
     ctx->pending_records = FLB_TRUE;
```

Comment:
please indent this line and the next one

Reply:
indentation is fixed